cmd/install-extensions/internal/run.go

```diff
@@ -0,0 +1,53 @@
+package internal
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func Run(environment map[string]string) error {
```

Comment:
suggestion: I think it would be nicer if the function for installing vscode extensions to just get a list of extensions names and 1 or 2 other paths you need in this function. It just seems a bit counter intuitive that for installing extensions you pass a dictionary.

IMO extracting stuff  from the env variables should happen in `cmd/install-extensions/main.go` and `cmd/internal/run.go` is only reponsible for installing the extensions - not both installing the extensions and extracting stuff from env vars.